airflow/providers/ssh/operators/ssh.py

```diff
@@ -61,9 +61,12 @@ class SSHOperator(BaseOperator):
     :param banner_timeout: timeout to wait for banner from the server in seconds
     """
 
-    template_fields: Sequence[str] = ('command', 'remote_host')
+    template_fields: Sequence[str] = ('command', 'environment', 'remote_host')
     template_ext: Sequence[str] = ('.sh',)
-    template_fields_renderers = {"command": "bash"}
+    template_fields_renderers = {
+        "command": "bash",
+        "environment": "json",
```

Comment:
Since `environment` is a Python dict, shouldn’t this be `python` instead?

Reply:
The values are later converted into strings *(as all environment variables)*.

Reply:
@uranusjr Done